client/signup/config/flows-pure.js

```diff
@@ -271,6 +271,16 @@ export function generateFlows( { getSiteDestination = noop, getPostsDestination
 		};
 	}
 
+	if ( config.isEnabled( 'signup/import-landing-handler' ) ) {
+		flows[ 'from-site' ] = {
+			steps: [ 'import-from-url', 'user', 'domains' ],
+			destination: ( { siteSlug } ) => `/settings/import/${ siteSlug }`,
+			description: 'A flow to kick off an import during signup',
+			disallowResume: true,
+			lastModified: '2018-08-05',
```

Comment:
## TODO 

bump this when landing

Reply:
I don't actually think this is used anywhere :).